oshi-core/src/main/java/oshi/software/os/unix/solaris/SolarisFileSystem.java

```diff
@@ -20,10 +20,7 @@
 import oshi.annotation.concurrent.ThreadSafe;
 import oshi.software.common.AbstractFileSystem;
 import oshi.software.os.OSFileStore;
-import oshi.util.ExecutingCommand;
-import oshi.util.FileSystemUtil;
-import oshi.util.Memoizer;
-import oshi.util.ParseUtil;
+import oshi.util.*;
```

Comment:
No star imports, please. I'm sure this has failed checkstyle somewhere in the CI tests below. :)